script.js

```diff
@@ -0,0 +1,55 @@
+let bookList = {};
+const displayBooks = document.querySelector('.all-books');
+const newTitle = document.querySelector('#title');
+const newAuthor = document.querySelector('#author');
+const addBook = document.querySelector('#btn');
+let StoreBooks = [];
+const ClearFields = () => {
+  newTitle.value = '';
+  newAuthor.value = '';
+};
+const DisplayAllBooks = () => {
+  ClearFields();
+  displayBooks.innerHTML = '';
+  StoreBooks = JSON.parse(localStorage.getItem('bookList')) || [];
+  StoreBooks.forEach((Books, index) => {
+    displayBooks.innerHTML += `
+        <div class="bookList">
+        <h1 class="titles">${Books.Title}</h1>
+        <h2 class="authors">${Books.Author}</h2>  
+        <button class="remove" id="${index}" onclick="RemoveBooks(event)">Remove</button>
+        <br>
+        <br>
+        <hr class="rule"> 
+        <br>
+        </div>`;
+  });
+};
+
+const saveBooks = () => {
+  addBook.addEventListener('click', () => {
+    if (newTitle.value === '' || newAuthor.value === '') return;
+    bookList = {
+      Title: newTitle.value,
+      Author: newAuthor.value,
+    };
+    StoreBooks.push(bookList);
+    localStorage.setItem('bookList', JSON.stringify(StoreBooks));
+    DisplayAllBooks();
+    ClearFields();
+  });
+};
+
+const RemoveBooks = (event) => {
+  const Findid = event.target.id;
+  const DeleteBook = StoreBooks.filter((Book, index) => {
+    if (index !== Findid) {
+      return Book;
+    }
+  });
+  localStorage.setItem('bookList', JSON.stringify(DeleteBook));
+  DisplayAllBooks();
+};
```

Comment:
- [ ] When I try to remove a book, nothing happens. The `Findid` variable is a string and not a number, so when you compare the loop `index` with `Findid` in your `if` statement using strict equality, it will always return true for every loop and no book will be deleted. I would advise you to convert the `Findid` to a number before you compare. 💻